backend/src/main/java/co/inventorsoft/scripty/service/SecurityService.java

```diff
@@ -6,17 +6,34 @@
 import org.springframework.stereotype.Service;
 
 import co.inventorsoft.scripty.exception.ApplicationException;
+import co.inventorsoft.scripty.model.entity.Project;
+import co.inventorsoft.scripty.repository.ProjectRepository;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.experimental.FieldDefaults;
 
 /**
  * @author lzabidovsky 
  */
 @Service
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
 public class SecurityService {
-	
+
+	ProjectRepository projectRepository;
+
 	public void authenticationHasRoleUser(Authentication authentication) {
 		if(!authentication.getAuthorities().contains(new SimpleGrantedAuthority("ROLE_USER"))) {
 			throw new ApplicationException(authentication.getName()+" does not have ROLE_USER" , HttpStatus.FORBIDDEN);
 		}
 	}
 
+	public void projectUserIsOwner(Long projectId, Authentication auth) {
+		Project project = projectRepository.findById(projectId).orElseThrow(() -> new ApplicationException("Project with ID="+projectId+" does not exist" , HttpStatus.NOT_FOUND));
```

Comment:
Please, pass project entity here as a parameter. Security service should care only about access control rules. it shouldn't know how to load business objects